src/components/RPCProvider.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { web3Connection } from '../reducers/web3';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import TopHeader from '../components/TopHeader';
+import { default as config } from '../config.js';
+
+const RPCConnectionError = ({ ...props }) => (
+  <div>
+    <TopHeader />
+    <Grid fluid>
+      <Row className="well well-error">
+        <Col md={12} >
+          <h1 className="center">RPC connection Error (502)</h1>
+          <p>Trying to connect to rpc node {config.rpcHost} received an invalid response.</p>
+          <a href="/" >Reload {props.counter}</a>
```

Comment:
here just block interface by modal and ask to reload the page, I think link is not necessary